src/GovUk.Education.ExploreEducationStatistics.Content.Services/PublicationService.cs

```diff
@@ -0,0 +1,109 @@
+            .CheckEntityExists<Publication>(query => query
+                .Include(p => p.Contact)
+                .Include(p => p.LegacyReleases)
+                .Include(p => p.Topic)
+                .ThenInclude(topic => topic.Theme)
+                .Where(p => p.Slug == publicationSlug))
+            .OnSuccess(async publication =>
+            {
+                var publicationViewModel = _mapper.Map<CachedPublicationViewModel>(publication);
+                // NOTE: BlobCache won't cache result if return a Either.IsLeft - if there is no latestRelease
+                return await GetLatestRelease(publication.Id)
+                    .OnSuccess(latestRelease =>
+                    {
+                        publicationViewModel.LatestReleaseId = latestRelease.Id;
+                        publicationViewModel.Releases = ListReleases(publication.Id);
+                        return publicationViewModel;
+                    });
+            });
+    }
+
+    private async Task<Either<ActionResult,Release>> GetLatestRelease(Guid publicationId)
+    {
+        // @MarkFix EES-3149 exclude superseded releases here
+        var releases = await _contentDbContext.Releases
+            .Include(r => r.Publication)
+            .Where(release => release.PublicationId == publicationId)
+            .ToListAsync();
+
+        var release = releases
+            .Where(IsLatestPublishedVersionOfRelease)
+            .OrderBy(release => release.Year)
+            .ThenBy(release => release.TimePeriodCoverage)
+            .LastOrDefault();
+
+        if (release == null)
+        {
+            return new NotFoundResult();
+        }
+
+        return release;
+    }
+
+    private List<ReleaseTitleViewModel> ListReleases(Guid publicationId)
+    {
+        var releases = _contentDbContext.Releases
+            .Include(r => r.Publication)
+            .Where(release => release.PublicationId == publicationId)
+            .ToList()
+            .Where(IsLatestPublishedVersionOfRelease)
+            .OrderByDescending(release => release.Year)
+            .ThenByDescending(release => release.TimePeriodCoverage);
+        return _mapper.Map<List<ReleaseTitleViewModel>>(releases);
+    }
+
+    private static bool IsLatestPublishedVersionOfRelease(Release release)
+    {
+        if (release.Publication?.Releases == null || !release.Publication.Releases.Any())
+        {
+            throw new ArgumentException(
+                "Release must be hydrated with Publications Releases to test the latest published version");
+        }
+
+        return
+            // Release itself must be live
+            release.Live
+            // It must also be the latest version unless the later version is a draft not included for publishing
+            && !release.Publication.Releases.Any(r =>
+                r.Live
+                && r.PreviousVersionId == release.Id
+                && r.Id != release.Id);
+    }
```

Comment:
I think you should try and reuse some of the logic in `Publication.cs` rather than duplicate this again here. It looks like adding a variation of `ListActiveReleases` like `GetPublishedReleases` which in turn uses `IsLatestPublishedVersionOfRelease` instead of  `IsLatestVersionOfRelease` would work.